renderer/actions/FieldActions.ts

```diff
@@ -12,20 +12,20 @@ export const updateHeart: FieldActions['updateHeart'] = () => ({
   type: consts.FieldActionsTypes.UPDATE_HEART,
 });
 
-export const updateMaster: FieldActions['updateMaster'] = (msg: any) => ({
+export const updateMaster: FieldActions['updateMaster'] = (msg: { blue: string; gold: string; }) => ({
```

Comment:
@ewc340 Tried to fix the any types, but a bit unsure if this is right. Please check.